### PR TITLE
Changes after moving monitoring app to monitoring namespace

### PR DESCRIPTION
**Pull request by UnderGreen**
+ fix some issues from linter output

Closes https://github.com/gravitational/gravity.e/issues/3748.

### CODE REVIEW

**Comment by r0mant** [comment]
@UnderGreen @a-palchikov I've updated this PR to determine monitoring namespace before forwarding request to remote cluster to preserve compatibility b/w updated Ops Center and older clusters. I've tested it in the following scenario: install 5.0 Ops Center, install 5.0 cluster via it, upgrade Ops Center to 5.2 -> monitoring tab for the cluster works.

Also addressed Dima's previous comments, so this should be good to go.

File: lib/clients/clients.go [hunk 1]
```diff
@@ -191,7 +196,51 @@ func (r *ClusterClients) newAppsClient(clusterName string) (app.Applications, er
...
+		TLSClientConfig: rest.TLSClientConfig{
+			CAData:   signResponse.CACert,
+			CertData: signResponse.Cert,
+			KeyData:  key,
+		},
+		WrapTransport: func(t http.RoundTripper) http.RoundTripper {
+			transport := t.(*http.Transport)
+			transport.DialContext = func(ctx context.Context, network, address string) (net.Conn, error) {
+				return remoteCluster.Dial(
+					&httplib.Addr{Net: "tcp", Addr: "127.0.0.1:3024"},
```
Comment (a-palchikov) [inline lib/clients/clients.go]: nit: Should this be in defines? I believe it's being used [here](https://github.com/gravitational/gravity/blob/sergei/monitoring_namespace/lib/webapi/forwarder.go#L198) and [here](https://github.com/gravitational/gravity/blob/sergei/monitoring_namespace/lib/webapi/forwarder.go#L316) as a literal as well

File: lib/ops/monitoring/monitoring.go [hunk 2]
```diff
@@ -33,3 +42,19 @@ type RetentionPolicy struct {
 	// Duration is the policy duration
 	Duration time.Duration `json:"duration"`
 }
+
+// GetNamespace uses the provided Kubernetes client to determine namespace
+// where monitoring resources reside
+func GetNamespace(client *kubernetes.Clientset) (string, error) {
```
Comment (a-palchikov) [inline lib/ops/monitoring/monitoring.go]: I would use the interface as minimal as possible (`CoreV1`):
```go
GetNamespace(client kubernetes.CoreV1Interafce) {...}
```

File: lib/clients/clients.go (hunk 1 above)
Comment (r0mant) [reply lib/clients/clients.go]: Yeah, good point. Wanted to make it a default but forgot.

**Pull request merged**